docs/source/community/persons_of_interest.rst

```diff
@@ -225,6 +225,11 @@ PowerPC
 
 -  Alfredo Mendoza (`avmgithub <https://github.com/avmgithub>`__)
 
+Docs / Tutorials
+~~~~~~~
```

Comment:
The underline needs to cover the entire word above it, otherwise you end up with build failures

```suggestion
Docs / Tutorials
~~~~~~~~~~~~~~~~
```